docs/sources/write/front-matter/index.md

```diff
@@ -288,7 +288,11 @@ cascade:
 
 <!-- vale Grafana.Headings = YES -->
 
-Use `menuTitle` to specify a different heading in the sidebar navigation than the `title` element; for example, if you want to abbreviate the topic heading in the table of contents.
+Use `menuTitle` to specify a different title in the sidebar navigation than in the `title` element.
+For example, if you want to abbreviate the topic title in the table of contents.
+
+Don't remove the verb from [task topic](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/) titles.
+The verb helps the reader know that they're navigating to a task topic before they follow the link.
```

Comment:
```suggestion
Don't remove the verb from the `title`.
The verb helps the reader know that they're navigating to a  [task topic](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/) before they follow the link.
```

What if we tweaked it a little more here. We want to keep the verb in the title metadata, correct? Which also then matches the H1 content.

Reply:
The verb should be in the `title` (and the first heading should match the `title`) AND it should also be in the `menuTitle` so that readers navigating via the table of contents menu know that they're navigating to a task topic before they follow the link.